src/main/java/com/example/demo/service/base/GeneralService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.demo.service.base;
+
+import com.example.demo.model.Team;
+import com.google.common.collect.Lists;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.data.repository.CrudRepository;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+
+@RequiredArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+public class GeneralService<ENTITY, ID extends Integer> {
+    CrudRepository<ENTITY, ID> repository;
+
+    public ENTITY getById(ID id) {
```

Comment:
`org.springframework.transaction.annotation.Transactional`
Should be used if you access DB.